libtransmission/quark.h

```diff
@@ -286,6 +286,7 @@ enum
     TR_KEY_recent_relocate_dir_4,
     TR_KEY_recheckProgress,
     TR_KEY_remote_session_enabled,
+    TR_KEY_remote_session_https,
```

Comment:
The new enum item seems to be out of order. Expect to see it after `TR_KEY_remote_session_host`, based on the sort order in `MyStatic` array.

Reply:
fixed.